api/nakadi-event-bus-api.yaml

```diff
@@ -1626,6 +1654,11 @@ definitions:
           Might assume the special name BEGIN, meaning a pointer to the offset of the oldest
           available event in the partition.
         type: string
+      unconsummed_events:
+        description: |
+          Number of events unconsummed by the client. This is also known as consumer lag and is used for monitoring
+          purposes by consumers interested in detecting the trend in growth of the number of unconsumed events.
```

Comment:
`detecting the trend in growth of the number of unconsumed events.` -> `keeping an eye on the number of unconsumed events.` or `monitoring the evolution of the number of unconsumed events.`